### PR TITLE
Implement rspec

### PR DESCRIPTION
**Pull request by MonikaNechkoska**
Task: https://www.notion.so/1c2ae719c5eb4b408218bc148d814f0f?v=91b9786c3ecb41f99a08200af517e7a6&p=44c68ce1c519447c984254ef6b38a7e6

### CODE REVIEW

**Comment by OrceMaceski** [comment]
You should implement **shoulda-matchers** as well because they complement rspec:
https://github.com/thoughtbot/shoulda-matchers

File: Gemfile [hunk 1]
```diff
@@ -34,9 +34,18 @@ gem 'rubocop', require: false
...
+end
+
 group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'rspec-rails', '~> 5.0.0'
+end
+
+group :test do
+  gem 'shoulda-matchers', '~> 5.0'
```
Comment (violeta-p) [inline Gemfile:48]: No need to duplicate groups for same environments. You can put: `gem 'rspec-rails', '~> 5.0.0'` and `gem 'shoulda-matchers', '~> 5.0'` in first `group :development, :test`.

**Review by violeta-p — COMMENTED** [review]
I'd suggest adding `factory bot` gem as well.

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../config/environment', __dir__)
+# Prevent database truncation if the environment is production
+abort('The Rails environment is running in production mode!') if Rails.env.production?
+require 'rspec/rails'
+require 'support/shoulda_matchers'
```
Comment (OrceMaceski) [inline spec/rails_helper.rb:10]: I believe directory structure has not been preserved here

**Review by OrceMaceski — APPROVED** [review]

**Review by OrceMaceski — APPROVED** [review]

**Pull request merged**